### PR TITLE
feat: adding calcutta default fonts for vnc

### PR DESCRIPTION
**Pull request by gavin-k-lee**
Fonts on the VNC have been updated to be consistent with the rest of the UI (Calcutta-Medium).

### CODE REVIEW

**Comment by gavin-k-lee** [comment]
Tested on https://renkulab.io/projects/lee.gavin.k/renku-desktop/environments which fetches the image built on GH actions with the updates:

image

Checked with some symbols as well to confirm font:

**Review by ciyer — CHANGES_REQUESTED** [review]
The main content looks consistent with the UI now! 🎉 

Could you 
image

But the "Refresh your browser..." text is in a different font (Helvetica for me, something else for you). Could you set that to use Calcutta as well?

**Comment by gavin-k-lee** [comment]
I believe that's fixed now (the same project was updated so you can view it there now)

image

File: docker/vnc/fonts/calcutta/Calcutta-Medium.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			font-family: "Calcutta-Medium", Georgia;
+		}
+	</style>
+</head>
+
+<body>
+	<p contenteditable="true">The quick brown fox jumps over the lazy dog. $123.45!</p>
+</body>
+
+</html>
```
Comment (ciyer) [inline docker/vnc/fonts/calcutta/Calcutta-Medium.html:21]: Do you want to delete this now that it has been tested?

**Review by ciyer — APPROVED** [review]
👍 Looks great now! Do you want to delete `Calcutta-Medium.html` now, or do you want to keep it around? I'm approving and leaving the decision on that file to you, Let me know if you need me to re-approve.

File: docker/vnc/fonts/calcutta/Calcutta-Medium.html (hunk 1 above)
Comment (gavin-k-lee) [reply docker/vnc/fonts/calcutta/Calcutta-Medium.html:21]: Yep let's delete it.

**Review by ciyer — APPROVED** [review]

**Pull request merged**